andela/team-odd-bn-backend
# # 169258627 Deployment of base application to heroku

#### What does this PR do?
Deploys the base application on Heroku

#### Description of Task to be completed?
- Set up Procfile 
- Set up the staging environment on Heroku
- Set up the production environment on Heroku


#### How should this be manually tested?
- Download postman
- Via postman, use the link `https://team-odd-bn-backend-staging.herokuapp.com/` to check the staging environment
- Via postman, use the link `https://team-odd-barefoot-backend.herokuapp.com/` to check the production environment

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#169258627](https://www.pivotaltracker.com/story/show/169258627)

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-11-08 at 15 06 13" src="https://user-images.githubusercontent.com/56693193/68481253-5cb85180-023f-11ea-94c2-255aee614f93.png">
<img width="1440" alt="Screen Shot 2019-11-08 at 15 23 57" src="https://user-images.githubusercontent.com/56693193/68481262-693caa00-023f-11ea-86d2-f96b1f008fdf.png">
